lms/templates/discussion/_discussion_module.html

```diff
@@ -4,9 +4,9 @@
 from django_comment_client.permissions import has_permission
 %>
 
-<div class="discussion-module" data-discussion-id="${discussion_id | h}">
+<div class="discussion-module" data-discussion-id="${discussion_id | h}" data-user-create-comment="${can_create_comment}" data-user-create-subcomment="${can_create_subcomment}">
```

Comment:
Why not call this data-can-create-comment and data-can-create-subcomment like the vars in both Python and JS code?